src/Playwright/Core/FormData.cs

```diff
@@ -51,13 +51,17 @@ private IFormData SetImpl(string name, object value)
 
     public IFormData Set(string name, FilePayload value) => SetImpl(name, value);
 
-    internal IList<object> ToProtocol()
+    internal IList<object> ToProtocol(bool throwWhenSerializingFilePayloads = false)
     {
         var output = new List<object>();
         foreach (var kvp in Values)
         {
             if (kvp.Value is FilePayload file)
             {
+                if (throwWhenSerializingFilePayloads)
+                {
+                    throw new PlaywrightException("Form requests don't support file payloads, use MultiPart=formData instead.");
```

Comment:
Should it be `Multipart` instead?

```suggestion
                    throw new PlaywrightException("Form requests don't support file payloads, use Multipart=formData instead.");
```

https://github.com/microsoft/playwright-dotnet/blob/657f415d3eaa05df2d3dbff0a56526aad285d6f1/src/Playwright/API/Generated/Options/APIRequestContextOptions.cs#L166-L167

Reply:
Good catch!